### PR TITLE
Fixes scaling issue `KeyError: 'x-lizzy-output'`

### PR DESCRIPTION
**Pull request by scherniavsky**
The issue is logged [here](https://tip.ci.zalan.do/job/sample-load-test/126/console)

### CODE REVIEW

File: lizzy_client/version.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 MAJOR_VERSION = 2
-MINOR_VERSION = 8
+MINOR_VERSION = 9
 REVISION = 3
```
Comment (jmcs) [inline lizzy_client/version.py]: please increase the revision (line 3)

Comment (jmcs) [reply lizzy_client/version.py]: leave minor version as 8

Comment (scherniavsky) [reply lizzy_client/version.py]: fixed, sorry for misunderstanding

**Comment by jmcs** [comment]
:+1:

**Review by jmcs — APPROVED** [review]

**Pull request merged**